meteor-app/package.json

```diff
@@ -4,7 +4,7 @@
   "private": false,
   "scripts": {
     "debug": "DEBUG=b2b:* MONGO_URL=mongodb://localhost:27017/b2b meteor --inspect --port 3030 --settings dev.settings.json",
-    "debug-local": "DEBUG=b2b:*  meteor --inspect --port 3030 --settings dev.settings.json",
+    "debug-local": "meteor --inspect --port 3030 --settings dev.settings.json",
```

Comment:
The setting of the environment variable `DEBUG` is important here, and you should not just delete it. 
Please have a look at https://www.npmjs.com/package/cross-env and see if we can use it to support both Windows and Mac users

Reply:
@saoudnazir Please also add this package to the project, and remove the number image files. Once done I can accept the PR and merge it (I will do these)

Reply:
@mikkelking Please have a look in Multi-Visits-Card. there is some error in there. It works on my machine but seems failed on circleci. I have removed number folder too.

Reply:
You need to update the snapshots. You can do it with this command: 
`npm run test:snapshots -- -u` and the commit/push

You also need to resolve the conflict in `meteor-app/package-lock.json`, you can do this by doing a `git checkout -f package-lock.json`